streams/mx.m3u

```diff
@@ -35,6 +35,8 @@ https://stream.oursnetworktv.com/latin/encoder74/playlist.m3u8
 https://s5.mexside.net:1936/medios/medios/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal4deMatehuala.mx",Canal 4 de Matehuala (360p) [Not 24/7]
 https://stream.oursnetworktv.com/latin/encoder203/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal5.mx",Canal 5
+http://streamsy.online:2999/happy2024/newyear/1457
```

Comment:
Xtream-Codes, not accepted.
![image](https://github.com/user-attachments/assets/0511a900-8755-4159-942b-eb58ecd0a9c0)